### ISSUE TITLE
Question-Answer Examples

### ISSUE THREAD

**Issue by Zenyker** [xenova/transformers.js#144]
Ca you please send us an example of question-answer please

**Comment by xenova** [comment]
Sure, here's some example code for it:

```js
import { pipeline, env} from '@xenova/transformers';

// You can remove this if you are running locally
env.allowLocalModels = false;

// Choose model to use
let model = 'Xenova/distilbert-base-uncased-distilled-squad';

// Specify question and context
let question = 'Who was Jim Henson?'
let context = 'Jim Henson was a nice puppet.'

// Run pipeline
let answerer = await pipeline('question-answering', model);
let outputs = await answerer(question, context);
console.log(outputs);
// {
//     "answer": "a nice puppet",
//     "score": 0.5768911502526741
// }
```

I'll add this example to the [docs](https://huggingface.co/docs/transformers.js/api/pipelines#module_pipelines.QuestionAnsweringPipeline).

**Comment by Zenyker** [comment]
thanks Bro.

**Comment by Zenyker** [comment]
Thanks Bro, but I receive this error, 
Error: Could not locate file: "https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english/resolve/main/tokenizer.json".

if I test it with other local model receive this

Server running at http://127.0.0.1:3000/
2023-06-09 16:58:36.008 node[64827:2142172] 2023-06-09 16:58:36.006837 [E:onnxruntime:, sequential_executor.cc:494 ExecuteKernel] Non-zero status code returned while running Add node. Name:'/embeddings/Add_1' Status Message: /Users/runner/work/1/s/onnxruntime/core/providers/cpu/math/element_wise_ops.h:560 void onnxruntime::BroadcastIterator::Append(ptrdiff_t, ptrdiff_t) axis == 1 || axis == largest was false. Attempting to broadcast an axis by a dimension other than 1. 512 by 1142
An error occurred during model execution: "Error: Non-zero status code returned while running Add node. Name:'/embeddings/Add_1' Status Message: /Users/runner/work/1/s/onnxruntime/core/providers/cpu/math/element_wise_ops.h:560 void onnxruntime::BroadcastIterator::Append(ptrdiff_t, ptrdiff_t) axis == 1 || axis == largest was false. Attempting to broadcast an axis by a dimension other than 1. 512 by 1142
".
Inputs given to model: {
  input_ids: Tensor {
    dims: [ 1, 1142 ],
    type: 'int64',
    data: BigInt64Array(1142) [
          4n,  1653n,  1538n, 1030n, 14774n,  1064n,     5n,  7714n,
      30931n,  1108n,  1480n, 1049n,  4899n, 16320n,  1456n,  3908n,
       1038n, 10332n,  1013n, 1287n,  1941n,  3339n,  1096n, 14774n,
       1049n, 28822n,  1017n, 1049n,  1260n,  1420n,  1042n,  1049n,
      10234n,  1009n,  1036n, 1040n, 13113n,  1017n,  1040n, 28822n,
       1341n,  1110n,  1040n, 2320n,  1795n,  1017n,  1040n,  1260n,
       1420n,  1110n,  1040n, 2740n,  1042n,  1040n, 10234n,  1110n,
       1040n,  2437n,  3762n, 1009n,  1365n, 26225n,  1062n,  8715n,
       1008n,  1079n,  3713n, 1036n,  3288n, 14710n,  1653n,  1538n,
       1079n,  1565n,  1009n,    3n,  1067n,  1038n, 20599n,  1008n,
       1153n,  1064n,  1354n, 6146n,  3460n,  1026n,  3677n,  1042n,
      30323n, 30935n, 11523n, 1017n,  1355n,  1467n,  1942n,  1847n,
       1049n, 10234n,  1009n, 1040n,
      ... 1042 more items
    ],
    size: 1142
  },
  attention_mask: Tensor {
    dims: [ 1, 1142 ],
    type: 'int64',
    data: BigInt64Array(1142) [
      1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n,
      1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n,
      1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n,
      1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n,
      1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n,
      1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n,
      1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n,
      1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n, 1n,
      1n, 1n, 1n, 1n,
      ... 1042 more items
    ],
    size: 1142
  },
  token_type_ids: Tensor {
    dims: [ 1, 1142 ],
    type: 'int64',
    data: BigInt64Array(1142) [
      0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n,
      0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n,
      0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n,
      0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n,
      0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n,
      0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n,
      0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n,
      0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n,
      0n, 0n, 0n, 0n,
      ... 1042 more items
    ],
    size: 1142
  }
}
/Users/greeksfarrier/Desktop/Source Code/AI/node_modules/@xenova/transformers/node_modules/onnxruntime-node/dist/backend.js:45
                    resolve(__classPrivateFieldGet(this, _OnnxruntimeSessionHandler_inferenceSession, "f").run(feeds, fetches, options));
                                                                                                           ^

Error: Non-zero status code returned while running Add node. Name:'/embeddings/Add_1' Status Message: /Users/runner/work/1/s/onnxruntime/core/providers/cpu/math/element_wise_ops.h:560 void onnxruntime::BroadcastIterator::Append(ptrdiff_t, ptrdiff_t) axis == 1 || axis == largest was false. Attempting to broadcast an axis by a dimension other than 1. 512 by 1142

    at /Users/greeksfarrier/Desktop/Source Code/AI/node_modules/@xenova/transformers/node_modules/onnxruntime-node/dist/backend.js:45:108
    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)
